src/bp/ui-studio/src/web/views/Nlu/entities/ListEntity.tsx

```diff
@@ -1,9 +1,9 @@
 import { Button, Colors, FormGroup, Icon, InputGroup, Position, Radio, RadioGroup, Tooltip } from '@blueprintjs/core'
 import { NLU } from 'botpress/sdk'
 import { lang, utils } from 'botpress/shared'
-import { toastFailure } from 'botpress/utils'
 import _ from 'lodash'
 import React, { useEffect, useState, useReducer } from 'react'
+import { toastFailure } from '~/components/Shared/Utils'
```

Comment:
Actually it should be `toast.failure` exported from `botpress.shared`. We should get rid of toasts in Shared/Utils

Reply:
fixed